### PR TITLE
Adding a row merger v2 performance test.

### PR DESCRIPTION
**Pull request by sduskis**

### CODE REVIEW

File: bigtable-hbase/src/test/java/com/google/cloud/bigtable/hbase/RowMergerPerf.java [hunk 1]
```diff
@@ -15,64 +15,45 @@
...
-
-    ReadRowsResponse response =
-        ReadRowsResponse.newBuilder().addChunks(contentChunk).addChunks(rowCompleteChunk)
-            .setRowKey(ByteString.copyFromUtf8(rowKey)).build();
-
-    responses.add(response);
-    return responses;
+    Column.Builder columnBuilder =
+        familyBuilder.addColumnsBuilder()
+        .setQualifier(ByteString.copyFromUtf8("Qaulifier"));
```
Comment (garye) [inline bigtable-hbase/src/test/java/com/google/cloud/bigtable/hbase/RowMergerPerf.java]: nit: typo

File: bigtable-hbase/src/test/java/com/google/cloud/bigtable/hbase/v2/RowMergerPerf.java [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+  private static List<ReadRowsResponse> createResponses() {
+
+    CellChunk contentChunk = CellChunk.newBuilder()
+        .setFamilyName(StringValue.newBuilder().setValue("Family1"))
+        .setQualifier(BytesValue.newBuilder().setValue(ByteString.copyFromUtf8("Qaulifier")))
+        .setRowKey(ByteString.copyFrom(Bytes.toBytes("rowkey-0")))
+        .setValue(ByteString.copyFrom(RandomStringUtils.randomAlphanumeric(10000).getBytes()))
+        .setTimestampMicros(0L)
+        .setCommitRow(true)
```
Comment (garye) [inline bigtable-hbase/src/test/java/com/google/cloud/bigtable/hbase/v2/RowMergerPerf.java]: Would be good to test multi-chunks rows, too.

**Comment by garye** [comment]
LGTM, just a couple minor things

File: bigtable-hbase/src/test/java/com/google/cloud/bigtable/hbase/v2/RowMergerPerf.java (hunk 2 above)
Comment (sduskis) [inline bigtable-hbase/src/test/java/com/google/cloud/bigtable/hbase/v2/RowMergerPerf.java]: I added a TODO for multi-chunks

**Pull request merged**